tests/test_blob_func.py

```diff
@@ -18,63 +18,75 @@ def areaToFill():
     rRowInds[rInd] = rRow
     rColInds[cInd] = rCol
     for i in range(3):
-        timg[i, rRowInds[0]: rRowInds[1], rColInds[0]: rColInds[1]] = 100
-    timg[-1, rRowInds[0]: rRowInds[1], rColInds[0]: rColInds[1]] = 255
+        timg[i, rRowInds[0] : rRowInds[1], rColInds[0] : rColInds[1]] = 100
```

Comment:
`black` is adding these, which `flake8` does not like. @perrygeo @sgillies @normanb @vincentsarago seen this before? Also is doing the same on a dummy file:

```python
an_indexable_list = list(range(10))

a = [1]
b = [3]
c = [5]


an_indexable_list[a[0]:b[0]]
an_indexable_list[b[0]:c[0]]
```

Is black formatted to
```python
an_indexable_list = list(range(10))

a = [1]
b = [3]
c = [5]


an_indexable_list[a[0] : b[0]]
an_indexable_list[b[0] : c[0]]
```
This only occurs when the indexing variable is itself indexed, ie this passes both:
```python
an_indexable_list = list(range(10))

a = 1
b = 3
c = 5


an_indexable_list[a:b]
an_indexable_list[b:c]
```